m4_sessions/1803-inning/group_5.md

```diff
@@ -2,12 +2,34 @@ Group 5
 
 Group Members:
 
+Bruce Chung
+Alex Barnes
+Nico Lewis
+
 Name of Session:
 
+Social Media and Fake News
+
 Session Description (Why should students come to this session):
 
+Social media is already the main source of news for this generation.  Unlike tranditional news outlets, social media allows news to propogate on attention alone without regard to journalistic accuracy.  This is becoming an increasingly widespread and serious problem, even to national security as evidenced by the proliferation of Russian Fake News on facebook during the 2016 Election.
```

Comment:
Great session!  There is an old fake news session in the drive that is also really good.

Reply:
https://github.com/turingschool/gear-up/blob/master/m4_sessions/1706-inning/Fake_news.md